src/_ctf-private/ctf-sign-up-banner/SignupBanner.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Button, Container, Flex, Text, useBreakpointValue, useTheme } from '@chakra-ui/react';
+import Link from 'next/link';
+
+import { useSticky } from './hooks/useSticky';
+
+import CfLogo from '@icons/cf-logo.svg';
+
+export const SignupBanner = () => {
+  const isMobile = useBreakpointValue({ base: true, md: false });
+  const theme = useTheme();
+  // const MotionButton = motion(Button, { forwardMotionProps: true });
+  const signupLink =
+    'https://www.contentful.com/starter-templates/ecommerce-website/sign-up/?action=create_starter_template&template_name=ecommerce';
+  const systemUIFontFamilies =
+    '"Avenir Next W01", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"';
+
+  const { sticky, stickyRef } = useSticky();
+
+  return (
+    <>
+      <Box
+        bg="#0033A3"
+        ref={stickyRef}
+        css={sticky && { position: 'fixed', top: 0, width: '100%', zIndex: 200 }}>
```

Comment:
Please don't use the `css` attribute. This will create a `<style>` tag in the `<head>` and is intended for specific usecases; if you want to use this syntax please use `sx`, it stands for `styled-components`.

What would be even better:

```ts
...
position="sticky"
top="0"
width="100%"
zIndex={1}
...
```